### PR TITLE
Gather

### PR DESCRIPTION
**Pull request by ashantiowusu-brafi**
test help:

Right now my text to be displayed is the datetime on Page 5 in gather.html

### CODE REVIEW

**Comment by ashantiowusu-brafi** [comment]
data[1].datetime

how could I create a function that checks if the index is a poem that is posted within a certain datetime property of 10 minutes?

File: frontend/js/firstPage.js [hunk 1]
```diff
@@ -61,8 +61,13 @@ function authorEntered (e) {
     datetime: Date.now(), // number, unicode timestamp
     location: [crd.latitude, crd.longitude], // Array of GPS coordinates
     text: poem, // string
+<<<<<<< HEAD
+    author, // string,
+    route: chosenRoute // number, a bus route
+=======
     author: author, // string,
     route: Number(chosenRoute) // number, a bus route
+>>>>>>> e6bc869222e3eb65379520b2ce0769ce959bf945
```
Comment (nbriz) [inline frontend/js/firstPage.js]: these are git makers, they show up in code when there's a merge conflict, u'll need to clean this up before the home page will work. replace this function with the following:

```js
function authorEntered (e) {
  const textField = document.querySelector('#textid')
  author = textField.value
  addBubble(author, 'bubble2')
  textField.value = ''
  postPoem({
    datetime: Date.now(), // number, unicode timestamp
    location: [crd.latitude, crd.longitude], // Array of GPS coordinates
    text: poem, // string
    author: author, // string,
    route: Number(chosenRoute) // number, a bus route
  })
}
```

that's the way the function looks on the class website

File: frontend/js/gather.js [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
 
-const url = 'package.json'
+const url = 'poems/poems.json'
 const req = { method: 'GET' }
 fetch(url, req)
   // turn json information into js object
   .then(res => res.text())
   // get data from poem database and print java script object "time"
   .then(data => {
     const obj = JSON.parse(data)
```
Comment (nbriz) [inline frontend/js/gather.js]: here's how we can filter the array the way u described:
```js
    const obj = JSON.parse(data)
    // the variable below is the value for 10 minutes in milliseconds
    const min = 1000 * 60 * 10
    // this one is a timestamp for 10mins ago
    const past = Date.now() - min
    // here we use the array's "fitler" method to creaate a new array "sel"
    // which contains only the objcts with a datetime greater than 10mins ago
    const sel = obj.filter(p => p.datetime > past)  
 ```
keep in mind that the poems in the `poem.json` have `datetimes` all of which are more than 10mins ago (which means they'll all get filtered out && the `sel` array will be empty), but u can test this by changing their datetime values to something more recent (u can enter `Date.now()` in the browser's JavaScript console to create a timestamp for the current time)

**Review by nbriz — CHANGES_REQUESTED** [review]
hey @ashantiowusu-brafi i noticed some git markers in the `firstPage.js` file, u'll want to fix that (see my comment below). i've also left an explanation for how to filter the array the way u described on slack

**Pull request merged**